rails/templates/default/database.yml.erb

```diff
@@ -13,5 +13,8 @@
 <% if @database[:port] -%>
   port: <%= @database[:port].to_i.inspect %>
 <% end -%>
+<% if @database[:schema_search_path] -%>
+  schema_search_path: <%= @database[:schema_search_path].to_s.inspect %>
```

Comment:
So I assume you want to convert the whatever-type to a String, and then output the string with double-quotes around it (hence the need for `.inspect`), right?